### PR TITLE
[AR] Translation of 03.md

### PR DESCRIPTION
**Pull request by aissam-out**

### CODE REVIEW

File: docs/ar/week03/03.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+| lang-ref | lang | title | translator |
+| ------ | ------ | ------ | ------ |
+| ch.03 | ar | الأسبوع 3 | [aissam out](https://github.com/aissam-out) |
```
Comment (Atcold) [inline docs/ar/week03/03.md:3]: **This won't work.**
Please, don't change the syntax of the `MD` file.
My script does not understand this format.

File: docs/ar/week03/03.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+| lang-ref | lang | title | translator |
+| ------ | ------ | ------ | ------ |
+| ch.03 | ar | الأسبوع 3 | [aissam out](https://github.com/aissam-out) |
+
+<div dir="rtl">
```
Comment (Atcold) [inline docs/ar/week03/03.md:5]: Remove this.
My script is reading the `MD` file and serving a `HTML`.
We'll fix alignment in the `HTML` page.

File: docs/ar/week03/03.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+| lang-ref | lang | title | translator |
+| ------ | ------ | ------ | ------ |
+| ch.03 | ar | الأسبوع 3 | [aissam out](https://github.com/aissam-out) |
+
+<div dir="rtl">
+
+<!-- Lecture part A
+We first see a visualization of a 6-layer neural network. Next we begin with the topic of Convolutions and Convolution Neural Networks (CNN). We review several types of parameter transformations in the context of CNNs and introduce the idea of a kernel, which is used to learn features in a hierarchical manner. Thereby allowing us to classify our input data which is the basic idea motivating the use of CNNs.-->
+
+## محاضرة الجزء أ
```
Comment (Atcold) [inline docs/ar/week03/03.md:10]: Please, use "A".

File: docs/ar/week03/03.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+<div dir="rtl">
+
+<!-- Lecture part A
+We first see a visualization of a 6-layer neural network. Next we begin with the topic of Convolutions and Convolution Neural Networks (CNN). We review several types of parameter transformations in the context of CNNs and introduce the idea of a kernel, which is used to learn features in a hierarchical manner. Thereby allowing us to classify our input data which is the basic idea motivating the use of CNNs.-->
+
+## محاضرة الجزء أ
+
+نعاين أولاً تصورًا لشبكة عصبية من 6 طبقات. بعد ذلك نبدأ بموضوع التلافيف والشبكات العصبية الالتفافية (CNN). نستعرض عدة أنواع من تحويلات المعاملات في سياق شبكات CNN بالإضافة إلى تقديم فكرة النواة ، والتي تُستخدم لتعلم الميزات بطريقة هرمية. الشيء الذي يسمح لنا بتصنيف البيانات الداخلة وهي الفكرة الأساسية التي تحفز استخدام شبكات CNN.
+
+## محاضرة الجزء ب
```
Comment (Atcold) [inline docs/ar/week03/03.md:14]: Please, use "B".

File: docs/ar/week03/03.md [hunk 5]
```diff
@@ -1,17 +1,16 @@
-| lang-ref | lang | title | translator |
-| ------ | ------ | ------ | ------ |
-| ch.03 | ar | الأسبوع 3 | [aissam out](https://github.com/aissam-out) |
-
-<div dir="rtl">
+---
+lang-ref: ch.03
+title: Week 3
```
Comment (Atcold) [inline docs/ar/week03/03.md:3]: Missing translation of the title.

Comment (Atcold) [reply docs/ar/week03/03.md:3]: Missing `lang: ar`.

Comment (Atcold) [reply docs/ar/week03/03.md:3]: Missing `translator:` and `translation-date:`.

File: docs/ar/week03/03.md [hunk 6]
```diff
@@ -1,17 +1,16 @@
...
+---
+lang-ref: ch.03
+title: Week 3
+---
 
 <!-- Lecture part A
 We first see a visualization of a 6-layer neural network. Next we begin with the topic of Convolutions and Convolution Neural Networks (CNN). We review several types of parameter transformations in the context of CNNs and introduce the idea of a kernel, which is used to learn features in a hierarchical manner. Thereby allowing us to classify our input data which is the basic idea motivating the use of CNNs.-->
 
-## محاضرة الجزء أ
+## محاضرة الجزء A
```
Comment (Atcold) [inline docs/ar/week03/03.md:9]: I know I've asked for this, but now I'm not too sure it's a good idea. See [this](https://dlsquad.slack.com/archives/C0193CG5VNX/p1598772792300600).
Any thoughts?

Comment (aissam-out) [reply docs/ar/week03/03.md]: Yeah ! I think it's better to use the arabic letters .. I made the changes you requested :) 👍

**Review by Atcold — APPROVED** [review]
I've edited `docs/_config.yml` for you.

**Pull request merged**